### ISSUE TITLE
Arctos SQL tables break/lose connection to VertNet IPT, cause failed publication event

### ISSUE THREAD

**Issue by dbloom** [ArctosDB/arctos#9378]
@dustymc @mkoo @ekrimmel This issue is posted in response to @bryansmclean and issue #9377 

This issue is not new for Arctos and has been occurring on a small scale since summer of 2025. I had written it off as the usual sort of dance between TACC and the IPT.  It started as just a few resources every month, but it's almost every resource, every month. Something changes in or around the SQL tables that the IPT queries on a monthly basis when each resource is scheduled to auto-publish. I've pinged Dusty about this at least once a month for several months to ask him to kick the server box and reset the tables from specific resources - and I usually only do this when I see a GH issue or get an email query. Dusty has dutifully performed said kick every time and then I republish manually and all is well, at least for a month.

The issue is that all resources are reduced in the occurrence and media tables - for occurrences the table is suddenly reduced from 91 to 0 columns. The same happens for media, just a different # of columns. When this happens, I have to notify Dusty, then he has to reset everything manually and then I have to go into the IPT to relink to the tables and then republish manually - for me that can take 5-10+ hours if I have to do it for every resource in published to the IPT.

Here is my IPT error:

<img width="1335" height="627" alt="Image" src="https://github.com/user-attachments/assets/807ee734-fbeb-4fe7-8f96-8c600cd578fe" />

It would be great to fix this, because none of us want to fix this monthly. Right now, every Arctos resource shows red for next publication, so it is a guarantee I'll have to reset everyone of them. Also looks like this has been going on for several months for some of these resources because their last publication dates go back as far as Aug and Sept of 2025.

<img width="1216" height="1163" alt="Image" src="https://github.com/user-attachments/assets/7275d9a0-8483-494c-a543-a62e3a155e49" />

I don't know if it is related, but I wonder if this has something to do with TACC's security changes and the resetting of everyone's log in credentials. (And by the way, I can no longer log in to Arctos because VertNet cannot be validated in their system as an certified, real-boy organization - I will now have switch everything over to UCB address and hope it works and hope that UCB continues to allow me to use that address. Not sure how people using personal email addresses (e.g., gmail) are able to access Arctos.).

Happy to discuss here or on a call.
